posthog/api/capture.py

```diff
@@ -270,6 +270,9 @@ def get_event(request):
     if error_response:
         return error_response
 
+    # At this point, we don't now which team_id we are working with, so unbind if set.
+    structlog.contextvars.unbind_contextvars("team_id")
```

Comment:
I had expected the middleware to have unbound the contextvars 😞 

Should this unbind be put at the very beginning of `get_event`?

Reply:
Looks like it would be up to us to configure something like that in the middleware.

I was thinking about calling `clear_context` which clears the entire thing instead of just unbinding `team_id`, but our middleware [is setting thread id and process id](https://github.com/PostHog/posthog/blob/master/posthog/settings/logs.py#L35) apparently, and I wouldn't want to clear those.

I did move the `unbind` call to the top as that would stop any logs during error responses that could be incorrectly set and cause some red-herrings.

Reply:
Yeah the threadid was added to be able to debug deadlocks we were seeing